view/src/bin/pviewd.rs

```diff
@@ -32,22 +35,38 @@ struct Opt {
     pd_port: u16,
 }
 
-#[derive(Debug, Subcommand)]
+#[derive(Debug, clap::Subcommand)]
 enum Command {
-    /// Initialize the view service with a full viewing key.
     Init {
         /// The full viewing key to initialize the view service with.
         full_viewing_key: String,
     },
     /// Start the view service.
-    Start {
+    #[clap(subcommand, display_order = 100)]
+    Start(StartCmd),
+}
+#[derive(Debug, clap::Subcommand)]
+pub enum StartCmd {
+    /// Initialize the view service without spend capability
+    View {
         /// Bind the view service to this host.
         #[clap(long, default_value = "127.0.0.1")]
         host: String,
         /// Bind the view gRPC server to this port.
         #[clap(long, default_value = "8081")]
         view_port: u16,
     },
+    /// Initialize the custody service with a seed phrase
+    Custody {
+        /// Bind the view service to this host.
+        #[clap(long, default_value = "127.0.0.1")]
+        host: String,
+        /// Bind the view gRPC server to this port.
+        #[clap(long, default_value = "8081")]
+        view_port: u16,
+        // The seed phrase providing spend capability
+        seed_phrase: String,
+    },
```

Comment:
Rather than being able to start it in two different modes, I think we should have a config file generated by `init` that either has custody configuration or not, and then on start, run the custody service based on whether the config value is present.

Reply:
It might be useful to start with a PR that adds a TOML-able `Config` structure; then we just have to change `init` to write an appropriate config and `start` to read it.